pkg/plugins/manager/sources/sources.go

```diff
@@ -5,19 +5,22 @@ import (
 	"path/filepath"
 
 	"github.com/grafana/grafana/pkg/plugins"
+	"github.com/grafana/grafana/pkg/plugins/config"
 	"github.com/grafana/grafana/pkg/plugins/log"
 	"github.com/grafana/grafana/pkg/setting"
 )
 
 type Service struct {
-	cfg *setting.Cfg
-	log log.Logger
+	cfg  *setting.Cfg
+	pCfg *config.PluginManagementCfg
```

Comment:
I guess we don't need this change?

Reply:
ah yes, those were leftovers from a previous iteration. I will remove them